src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -320,8 +324,90 @@ def list_protectable_items(client, resource_group_name, vault_name, workload_typ
     return paged_items
 
 
+def fetch_tier(paged_recovery_points):
```

Comment:
private function naming should start with underscore.

Reply:
done